scripts/gulp/tasks/docs.ts

```diff
@@ -122,7 +122,14 @@ task('build:docs:schema', () =>
 );
 
 task('build:docs:webpack', cb => {
-  webpackPlugin(require('../../webpack/webpack.config').default, cb);
+  const webpackConfig = require('../../webpack/webpack.config').default;
+
+  const nightlyBuildEnvVar = {
+    'process.env.NIGHTLYRELEASEDATE': JSON.stringify(process.env.NIGHTLYRELEASEDATE),
```

Comment:
Shouldn't this override be defined in `scripts/webpack/webpack.config.ts` itself, using the same logic?

Reply:
Yes, that's a good point, probably a cleaner solution since the other env variables might also be declared there

Reply:
@ecraig12345 I made this change, could you take another look on this PR?